libs/db/src/schema.prisma

```diff
@@ -115,8 +115,11 @@ model cas__user {
   first_name                String?                     @db.VarChar(100)
   last_name                 String?                     @db.VarChar(100)
   avatar_url                String?
+  credit                    Int
+  credit_alert_lower_then   Int?
```

Comment:
**question:** Will it be hard to have multiple thresholds if we allow users to have multiple triggers? Now I'd say let's keep it simple, it's just a thought.

Reply:
It can be refactored in future.